packages/pharos-site/static/guidelines/text-input.docs.mdx

```diff
@@ -47,8 +44,8 @@ import BestPractices from '../../src/components/statics/BestPractices.tsx';
   <PageSection subSectionLevel={1} title="Labels">
     <ul>
       <li>
-        The input label needs to be helpful in telling the user what is expected of them, but they
-        do not need to be instructive
+        The input label needs to help tell the user what is expected of them, but they don't need to
```

Comment:
Suggestion is for grammar, but I think this could also be further clarified to indicate the nuance between "tell"/"help tell" and "instruct":

```suggestion
        The input label needs to tell the user what is expected of them, but it doesn't need to
```